### PR TITLE
Add majority of mintel-mixins

### PR DESCRIPTION
**Pull request by nabadger**
- Add a lot more mixnis
- Add a helper script (see updated readme and ./scripts)
- Split rules and alerts into separate dirs

I've temporarily disabled the runbook validation checks, since the new alerts we imported don't have them.

We'll need todo that work - but as a different case.

### CODE REVIEW

File: image-service.rules.libsonnet [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (primeroz) [inline image-service.rules.libsonnet]: duplicate file in the root ?

File: lib/mintel/rules/haproxy-ingress.libsonnet [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'haproxy-ingress.rules',
+        rules: [
+          {
+            expr: 'label_replace(\n  label_replace(http_backend_response_wait_seconds_bucket{backend!~"(error|stats|httpback|upstream).*"}, "mintel_com_service", "$1", "backend", "(.*)-\\\\d+$")\n    * on(mintel_com_service) group_left(label_app_kubernetes_io_owner)\n    label_join(kube_service_labels, "mintel_com_service", "-", "namespace", "service"),\n  "label_app_kubernetes_io_owner",\n  "satoshi",\n  "label_app_kubernetes_io_owner",\n  ""\n)\n',
```
Comment (primeroz) [inline lib/mintel/rules/haproxy-ingress.libsonnet]: i think those rules with escaped "\n" and similar are bad and will be hard to edit going forward.

I don't think they need to be, expr can be multiline right ?

Comment (nabadger) [reply lib/mintel/rules/haproxy-ingress.libsonnet]: yeah, let me fix the haproxy ones

Comment (nabadger) [reply lib/mintel/rules/haproxy-ingress.libsonnet]: Should be good now. Also replaced "\\\\d" with "\\d"

**Pull request merged**